app/models.py

```diff
@@ -193,3 +195,21 @@ def __repr__(self):
         if self.blacklisted:
             tags = ' BLACKLISTED'
         return f"<Key email={self.email} apikey={self.apikey}{tags}>"
+
+
+class VoteInformation(db.Model):
+    voter_apikey = db.Column(db.String, db.ForeignKey('key.apikey'), primary_key=True)
+    resource_id = db.Column(db.Integer, db.ForeignKey('resource.id'), primary_key=True)
+    current_direction = db.Column(db.String, nullable=False)
+    resource = db.relationship('Resource', back_populates='voters')
+    voter = db.relationship('Key', back_populates='voted_resources')
+
+    @property
+    def serialize(self):
+        return {
+                'vote_apikey': self.voter_apikey,
```

Comment:
Maybe this is paranoia, but I feel like we shouldn't be serializing the API key itself because if this were to be logged anywhere, we don't want to log API keys in plaintext... but as far as I can tell, we're not using this serialize function anywhere at this time, right?

I think anyone choosing to use the serialize function in the future would look at it and know what date it contains and therefore be careful about what they do with that serialized data. I don't know, I'm just not sure.

Do you have a strong opinion on this one way or the other? If not, maybe we should just take it out and if we ever need the API key itself during serialization we can add it back in at that time?

Reply:
The API key is also logged in the `voter` property, which I'm pretty sure we need.. The serialized info for the usage of this in the test looks like this:
```
{'vote_apikey': '1234abcd', 'resource_id': 1, 'current_direction': 'upvote', 'voter': <Key email=test@example.org apikey=1234abcd>, 'resource': <Resource 
        Name: name
        Languages: [<Language language>]
        Category: <Category Category>
        URL: https://resource.url
>}
```
So maybe we just leave both in there and it's fine

Reply:
Or maybe we should take the API key out of the serialize function for API key and just log the email address of the user? Why would we ever want to serialize the key itself? That just seems like a data leak waiting to happen...

I keep going back and forth, I'm open to suggestion on this

Reply:
Okay. It's been a while since I worked on it and I can't remember the reason behind the decision to add serialize function...But I understand the concern here. Let me know if this could be an issue later on and I can circle back to it.

Reply:
Well the reason is because if you do something like `print(vote_info)`, you get something rather unhelpful like
```
<VoteInformation (transient 139740452757904)>
```

Since `transient 139740452757904` tells me nothing about the specific VoteInformation I'm logging, I instead use the `.serialize` function to create a dictionary out of it.

In the case of `Resources` and `Languages`, it also provides a simple way to turn the model into a dictionary so we can return all that data in responses. I think we use that same pattern with the API key itself, when we give the API key to the user via the `/apikey` endpoint.

So, come to think of it, since we're not returning the data within a VoteInformation in a response, I think we probably don't need the serialize function at all. Let's just go ahead and take it out for now and if we need it for some reason in the future we can add it back in. Does that sound good?

Reply:
Yeah. Sounds good. Then I'll probably have to take out test for VoteInformation Model as well, right?

Reply:
You don't have to take it out, you just have to assert on each individual property rather than on the serialize